modules/veteran/app/workers/veteran/vso_reloader.rb

```diff
@@ -3,54 +3,58 @@
 require 'sidekiq'
 
 module Veteran
-  class VsoReloader
-    include Sidekiq::Worker
-    BASE_URL = 'https://www.va.gov/ogc/apps/accreditation/'
-
+  class VsoReloader < BaseReloader
     def perform
       array_of_organizations = reload_representatives
-      Veteran::Service::Organization.import(array_of_organizations, on_duplicate_key_ignore: true)
+      Veteran::Service::Representative.where.not(representative_id: array_of_organizations).destroy_all
     end
 
     private
 
     def reload_representatives
-      data = fetch_data('orgsexcellist.asp')
-      data.each do |hash|
-        find_or_create_representative(hash)
+      vso_reps = []
+      vso_orgs = fetch_data('orgsexcellist.asp').map do |hash|
```

Comment:
`hash` is a weak variable name.. can you use something that better reveals intention?

Reply:
I'll change it to something else like vso...

Reply:
`hash` is used in a few other places in here

Reply:
Done on all accounts.